snippets/base/migrations/0050_auto_20180925_1131.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.1 on 2018-09-25 11:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0049_auto_20180925_1059'),
+    ]
+
+    operations = [
```

Comment:
Any idea why this migration is here? I don't see the corresponding change to the model in this PR.

Reply:
Good question: I changes the repr values of `300` and `400` in another commit and forgot to trigger a migration (b/c why does this need a db migration?). So this is valid and it doesn't really belong to this PR but I had to sneak it in.

Reply:
Removed the migration from this PR to keep it clean and tidy

Reply:
Ah. Cool. As long as it's here on purpose it's fine with me :) It is odd that small changes like this that aren't reflected in the DB schema need migrations...